AncientReligions/events/ACR_egyptian_events.txt

```diff
@@ -329,4 +330,319 @@ narrative_event = {
 		}
 		name = "EVTOPTA_ancrel_4008"
 	}
-}
\ No newline at end of file
+}
+
+#######################
+# Sed Festival events #
+#######################
+
+#Sed festival 1st day: the offerings
+character_event = {
+	id = ancrel.4010
+	desc = "EVTDESC_ancrel_4010"
+	picture = "GFX_evt_temple_egyptian_pagan"
+	
+	is_triggered_only = yes
+	
+	immediate = {
+		
+	}	
+
+	option = {
+		name = "EVTOPTA_ancrel_4010" # OK
+		hidden_tooltip = {
+			random_list = {
+				95 = {
+					character_event = { id = ancrel.4011 days = 1 }
+				}
+				5 = {
+					modifier = {
+						factor = 3
+						trait = drunkard
+					}
+					modifier = {
+						factor = 3
+						trait = gluttonous 
+					}
+					modifier = {
+						factor = 0.3
+						trait = zealous
+					}
+					modifier = {
+						factor = 0.3
+						trait = patient
+					}
+					character_event = { id = ancrel.4014 days = 1 }
+				}
+			}
+		}
+	}
+}
+
+#Sed festival 2nd day: the race
+character_event = {
+	id = ancrel.4011
+	desc = "EVTDESC_ancrel_4011"
+	picture = "GFX_evt_temple_egyptian_pagan"
+	
+	is_triggered_only = yes
+	
+	immediate = {
```

Comment:
You have a couple of these empty immediate clauses, are they necessary to keep Validator quiet?